src/index.tsx

```diff
@@ -20,34 +20,36 @@
 
 // window.__deps = dependencies
 
-import React, { Component } from "react"
-import { render } from "react-dom"
-import { Grommet, ResponsiveContext } from "grommet"
-import { theme } from "./theme"
-import { IntlProvider } from 'react-intl';
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
-import { Dashboard, HelloWorld } from "./components"
+import React, { Component } from "react";
+import { render } from "react-dom";
+import { Grommet, ResponsiveContext } from "grommet";
+import { theme } from "./theme";
+import { IntlProvider } from "react-intl";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { Dashboard, HelloWorld } from "./components";
+import { CustomerListView } from "./views/customer/app";
 
 class AppBody extends Component {
-  static contextType = ResponsiveContext
+  static contextType = ResponsiveContext;
 
   render() {
     return (
       <Switch>
-        <Route exact path='/' component={Dashboard} />
-        <Route exact path='/hello' component={HelloWorld} />
+        <Route exact path="/" component={Dashboard} />
+        <Route exact path="/hello" component={HelloWorld} />
+        <Route exact path="/customers" component={CustomerListView} />
       </Switch>
-    )
+    );
   }
 }
 
 function loadLocaleData(locale: string) {
   switch (locale) {
-    case 'sk':
-    case 'sk-SK':
-      return import('./compiled-lang/sk.json');
+    case "sk":
```

Comment:
do not change formatting when you are adding a feature (in future there are will be merge conflicts...)